### PR TITLE
Fix mounting of ditto log files on host

### PR DESCRIPTION
**Pull request by stmaute**
Extended docker-compose.yml with a volume to be able to access the ditto log files on the host system. 
Same for ditto-cluster.yaml.

### CODE REVIEW

File: deployment/docker/docker-compose.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ services:
        TZ: Europe/Berlin
 
   policies:
-    image: docker.io/eclipse/ditto-policies:${DITTO_VERSION:-latest}
+    image: thingsdev.azurecr.io/eclipse-ditto/policies:${DITTO_VERSION:-latest}
```
Comment (thjaeckle) [inline deployment/docker/docker-compose.yml:28]: @stmaute this shouldn't be in here .. please change back

File: gateway/service/src/main/resources/logback.xml [hunk 2]
```diff
@@ -49,15 +49,15 @@
                 <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
                     <level>info</level>
                 </filter>
-                <file>connectivity.log</file>
+                <file>/var/log/ditto/gateway.log</file>
```
Comment (thjaeckle) [inline gateway/service/src/main/resources/logback.xml:52]: Oops, good catch 👍

**Review by thjaeckle — CHANGES_REQUESTED** [review]
See comment - the image location changed from the official docker image to our internal development URL

File: deployment/docker/docker-compose.yml (hunk 1 above)
Comment (stmaute) [reply deployment/docker/docker-compose.yml:28]: No this shouldn't be here. I will fix it. Good catch.

**Review by thjaeckle — APPROVED** [review]

File: deployment/kubernetes/deploymentFiles/ditto/ditto-cluster.yaml [hunk 3]
```diff
@@ -435,13 +435,13 @@ spec:
         imagePullPolicy: IfNotPresent
         args: ["java", "-jar", "starter.jar"]
         # Resource can be managed by setting the following resource config.
-        resources:
-          requests:
-            cpu: "1"
-            memory: "1Gi"
```
Comment (thjaeckle) [inline deployment/kubernetes/deploymentFiles/ditto/ditto-cluster.yaml:429]: 1Gi is IMO too much for policies as request

File: deployment/kubernetes/deploymentFiles/ditto/ditto-cluster.yaml [hunk 4]
```diff
@@ -233,13 +233,13 @@ spec:
...
```
Comment (thjaeckle) [inline deployment/kubernetes/deploymentFiles/ditto/ditto-cluster.yaml:233]: 512Mi should suffice

File: deployment/kubernetes/deploymentFiles/ditto/ditto-cluster.yaml [hunk 5]
```diff
@@ -132,13 +132,13 @@ spec:
...
```
Comment (thjaeckle) [inline deployment/kubernetes/deploymentFiles/ditto/ditto-cluster.yaml:134]: 512Mi should suffice

**Comment by thjaeckle** [comment]
LGTM 👍

**Pull request merged**